tests/packagedcode/data/buildpack/paketo-buildpacks/java-memory-assistant/expectedbuildpack.json

```
@@ -0,0 +1,112 @@
+{
+  "packages": [],
+  "dependencies": [],
+  "files": [
+    {
+      "path": "buildpack.toml",
+      "type": "file",
+      "package_data": [
+        {
+          "type": "generic",
+          "namespace": null,
+          "name": "java-memory-assistant",
+          "version": "{{.version}}",
```

```suggestion
          "version": null,
```
We should not keep a version value if we cannot extract this properly from somewhere in the manifest or elsewhere.